pkg/scheduler/api/v1/types.go

```diff
@@ -145,7 +145,7 @@ type UtilizationShapePoint struct {
 // ResourceSpec represents single resource and weight for bin packing of priority RequestedToCapacityRatioArguments.
 type ResourceSpec struct {
 	// Name of the resource to be managed by RequestedToCapacityRatio function.
-	Name apiv1.ResourceName `json:"name,casttype=ResourceName"`
+	Name apiv1.ResourceName `json:"name" protbuf:"casttype=ResourceName"`
```

Comment:
not sure what this change is about... the scheduler config API isn't proto-serialized (and "protbuf" is a typo anyway)

Reply:
possibly just drop the casttype directive?

Reply:
Good catch on the typo! Dropped the casttype directive entirely instead.

Reply:
@liggitt is right, we shoudn't have `protbuf`. Interestingly, when I first reviewed the PR, I saw only "casttype=xyz" was removed.

Reply:
@Huang-Wei that’s because this second `casttype` instance wasn’t there in the initial version of this PR. It was introduced in https://github.com/kubernetes/kubernetes/pull/77688 which got in through a rebase, but I fixed it differently than the first `casttype` instance (my bad!).